README.md

```diff
@@ -1,45 +1,48 @@
 # Programming with Cryptol and SAW
```

Comment:
Will you revert these changes to the README.md since you already have a pull request open for them?